### PR TITLE
fix ebtables non-filter table functionality

### PR DESCRIPTION
**Pull request by atonkyra**
ebtables-atomic introduced a regression which causes non-filter tables to stop working, this PR re-introduces support for broute and nat tables :)

In the original PR I hadn't considered that ebtables atomic functionality needs file for each table...

### CODE REVIEW

**Comment by MaxKellermann** [comment]
This PR contains text which is not present in the commit message.
Please explain which ebtables version broke this, and where this change is documented.

File: src/ferm [hunk 1]
```diff
@@ -2765,17 +2771,28 @@ sub execute_slow($$) {
...
+        }
     }
 
     my $status;
     while (my ($table, $table_info) = each %{$domain_info->{tables}}) {
         my $table_cmd = "$domain_cmd -t $table";
 
+        if ($domain eq 'eb') {
+            if ($table eq '') {
+                print "-->|$table|<--\n";
```
Comment (MaxKellermann) [inline src/ferm]: What for?

Comment (atonkyra) [reply src/ferm]: Oops, that is leftover from debugging. I updated the PR.

**Comment by atonkyra** [comment]
I updated the commit message and pushed test fixes I forgot to include.

**Comment by MaxKellermann** [comment]
OK, so it was not a change in the `ebtables` tool, but a bug in your own ferm code submission from #44...

**Comment by MaxKellermann** [comment]
The unit test fails.
(I have repaired Travis-CI recently, so its results should be reliable now.)

**Comment by atonkyra** [comment]
Rebased the changes against master and tests should be OK now.

**Comment by MaxKellermann** [comment]
Damn, I released 2.5.1 at the same time!

**Pull request merged**